src/main/java/github/scarsz/discordsrv/listeners/DiscordChatListener.java

```diff
@@ -105,17 +106,37 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
             DiscordSRV.debug("DiscordGuildMessagePreProcessEvent was cancelled, message send aborted");
             return;
         }
-
+        
+        List<Role> whitelistedRoles = new ArrayList<>();
+        // if we have a whitelist in the config
+        if (DiscordSRV.config().getBoolean("DiscordChatChannelRolesSelectionAsWhitelist")) {
+            whitelistedRoles = event.getMember().getRoles().stream()
+                               .filter(role -> DiscordSRV.config().getStringList("DiscordChatChannelRolesSelection").contains(DiscordUtil.getRoleName(role)))
+                               .collect(Collectors.toList());
+            // System.out.println(whitelistedRoles);
```

Comment:
Remove this line

Reply:
Line removed, it was me being too much in a hurry